### DISCUSSION TITLE
PlaytimeReward Placeholder (Remaining Rewards) + /cmi prewards playername claim 1-100 command

### DISCUSSION THREAD

**Discussion by AnimeDrache** [Zrips/CMI#8012] [Enhancement]
We could really need a placeholder that shows that the reward is ready to be claimed or how many rewards are ready to be claimed.

We could also need a command that allows to claim the reward through chat command cause bedrock players can't really click on chat. So something like "/cmi prewards [playerName] claim NUMBER/NAME-OF-REWARD"

**Comment by Zrips** [comment]
There actually is an option to claim rewards via command, will need to include its usage in default locale. Usage would be something like **/cmi prewards -c:rewardName** player name can be provided too if you want to extract reward for someone else.
I can add %cmi_user_prewards_count% placeholder to get number of rewards you can claim.

**Comment by AnimeDrache** [comment]
Would be awesome! Then I can ridd of our current reward plugin! Please let me know when its done! =)

**Comment by AnimeDrache** [comment]
Any new updates to this? =D can't wait to use it finally!
